h/views.py

```diff
@@ -39,9 +39,14 @@ def annotation(context, request):
     alternate = request.resource_url(request.root, 'api', 'annotations',
                                      context['id'])
 
+    description = context.get('text')
+    if len(description) > 200:
+        description = description[:200] + '...'
```

Comment:
Actually, I wouldn't add ellipses. Let's just try to split at a sentence boundary. I'm okay with that being really naive.